oak-auth-external/src/test/java/org/apache/jackrabbit/oak/spi/security/authentication/external/basic/DefaultSyncContextTest.java

```diff
@@ -1459,7 +1459,7 @@ public void testCreateValuesEmptyCollection() throws Exception {
 
     @Test
     public void testCreateValuesSkipsNull() throws Exception {
-        List<String> strings = Lists.newArrayList("s", null, null, "t");
+        List<String> strings = new ArrayList<>(Arrays.asList("s", null, null, "t"));
```

Comment:
Why not just `List.of("s", null, null, "t")` ?

Reply:
Because "List.of" does not suppoet null. See https://issues.apache.org/jira/browse/OAK-11239